### PR TITLE
sdjournal: add NewJournalFromFiles

### PR DESCRIPTION
**Pull request by msackman**
Needed sd_journal_open_files. So added it.

### CODE REVIEW

**Comment by msackman** [comment]
Well that's not fair! 503 when travis is trying to download stuff from debian stretch. Not even my code!

**Comment by lucab** [comment]
ok to test

File: sdjournal/journal.go [hunk 1]
```diff
@@ -426,6 +435,33 @@ func NewJournalFromDir(path string) (j *Journal, err error) {
 	return j, nil
 }
 
+// NewJournalFromFiles returns a new Journal instance pointing to a journals residing
+// in a given files. The supplied path may be relative or absolute; if
+// relative, it will be converted to an absolute path before being opened.
```
Comment (lucab) [inline sdjournal/journal.go]: Where is this abs() applied? I didn't find it neither here nor mentioned in the `sd_journal_open` manpage.

**Comment by lucab** [comment]
Thanks for this PR, it looks fine overall. I just have a minor doubt on the documented behavior. I re-triggered travis and it is green now. I also slightly amended the PR title, it would be nice to re-word the commit title as well.

**Comment by msackman** [comment]
Thanks for the review. Yes, I was puzzled about the comment in the other function too but I'm afraid I just copied that over. Looking at `man 2 open`, it looks like libc actually copes with relative paths by prefixing $CWD. So I suppose the doc/comment is just referring to that behaviour.

**Comment by lucab** [comment]
Then it looks like we are documenting some behavior which is actually an internal implementation several layers down. I suggest to either drop that part or to ensure that behavior by doing a conditional cwd-prefixing ourself.

**Comment by msackman** [comment]
Perfectly agree. I'll remove both those comments in the docs then - will get to that early tomorrow. Cheers.

**Comment by msackman** [comment]
Heh, same travis failure as before.

**Comment by msackman** [comment]
Many thanks.

**Pull request merged**